pkgs/tools/misc/pmbootstrap/default.nix

```diff
@@ -1,26 +1,26 @@
 { stdenv, lib, git, openssl, makeWrapper, buildPythonApplication, pytestCheckHook, ps
-, fetchPypi, fetchFromGitLab }:
+, fetchPypi, fetchFromGitLab, sudo }:
 
 buildPythonApplication rec {
   pname = "pmbootstrap";
-  version = "1.39.0";
+  version = "1.40.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "13krdb0xmwgldfn8izm1x3j6i6s4n9j8n8n0p4g5kd74ygkvfczz";
+    sha256 = "sha256-b/9NEURp42d/j/Fk8NUS0ZAG99q56eg0pEU/xkFnvrM=";
   };
 
   repo = fetchFromGitLab {
     domain = "gitlab.com";
     owner = "postmarketOS";
     repo = pname;
     rev = version;
-    sha256 = "0mrhgi5y3jwd922cmm2niy9ia0g8p6bsqjjxg523yandshpa3683";
+    sha256 = "sha256-2yyHAHoIlwHX2+LbwwK7AGrBDZlfkhtCcKAKHdJMBdQ=";
   };
 
   pmb_test = "${repo}/test";
 
-  checkInputs = [ pytestCheckHook git openssl ps ];
+  checkInputs = [ pytestCheckHook git openssl ps sudo ];
```

Comment:
sudo is most likely not working as intended here. It has no suid bit in the nix store.

Reply:
If I drop `sudo` the test will produce a lot of errors :( 
```
============================= test session starts ==============================
platform linux -- Python 3.9.6, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
rootdir: /build/pmbootstrap-1.40.0
collected 0 items / 41 errors   
```
Otherwise it will run fine considering the list of disabled tests